math-libs/pre_hook_rocPRIM.cmake

```
@@ -0,0 +1,7 @@
+# Disable INT128 support on Windows to avoid cross linkage issues when building
+# with clang but depending on MSVC built support libraries:
+# https://github.com/ROCm/TheRock/issues/405
+if(CMAKE_SYSTEM_NAME STREQUAL "Windows")
```

Would
```suggestion
if(WIN32)
```
work here? That's what Scott used in most places.